docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
   web:
     build: .
     container_name: feedadoc_web
-    command: bash -c "bundle && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+    command: tail -f /dev/null
```

Comment:
What is this doing?

Reply:
When the Docker command was the rails server, terminating the server would stop the container. By switching to this, we can use scripts to start/stop/restart. 

The issue this solved for me was the fact that emails wouldn't reload the code (and I didn't yet know about rails email preview).